python/core/qgsdiagramrendererv2.sip

```diff
@@ -94,6 +94,9 @@ class QgsDiagramSettings
 
     void readXML( const QDomElement& elem, const QgsVectorLayer* layer );
     void writeXML( QDomElement& rendererElem, QDomDocument& doc, const QgsVectorLayer* layer ) const;
+
+    //! @note caller is responsible for deletion of QStandardItems
+    QList< QStandardItem * > legendItems() const;
```

Comment:
That's correct. Also it would be useful to add notes that the method was added in 2.10